ReportingServicesTools/Functions/Admin/Set-RsDatabase.ps1

```diff
@@ -68,42 +88,52 @@ function Set-RsDatabase
 
         [switch]
         $IsRemoteDatabaseServer,
-        
+
         [Parameter(Mandatory = $True)]
         [Alias('DatabaseName')]
         [string]
         $Name,
 
         [switch]
         $IsExistingDatabase,
-        
+
+        [switch]
+        $HasDatabaseUserRights,
```

Comment:
remove this parameter please (see my other comment about the right scripts being idempotent)

Reply:
Updated